bin/packages/watch.js

```diff
@@ -24,7 +24,7 @@ const exists = ( filename ) => {
 
 // Exclude deceitful source-like files, such as editor swap files.
 const isSourceFile = ( filename ) => {
-	return /.\.js$/.test( filename );
+	return /.\.(js|scss)$/.test( filename );
```

Comment:
Not sure if we'd need some kind of rule excluding the components package scss from being watched, since that will end up being watched by both webpack and this script.

Reply:
I think it's fine :) it will generate a new CSS for the package but not use it in the webpack build.